Source/TokyoMirage_Resolution/rules.txt

```diff
@@ -0,0 +1,201 @@
+[Definition]
+titleIds = 00050000101ED700,00050000101ED800,0005000010131D00
+name = Resolution
+path = "Tokyo Mirage Sessions FE/Graphics/Resolution " 
+description = Changes the resolution of the game.
+version = 3
+
+#[TextureRedefine] #lod override, shimmer and possible sharper textures
+#formats  = 0x01a,0x31,0x32,0x33,0x34,0x35,0x431,0x432,0x433,0x434,0x435
+#overwriteRelativeLodBias = -4 #
+
+[Preset]
```

Comment:
Yes, but why default to 256x144 res? (help my graphics looks like crap)
Preferably I'd say start with 1080, or at least unscaled..  (help my graphics pack doesn't work)
hence the pre-set.  

[Preset]
name = 256x144
$width = 256
$height = 144
$gameWidth = 1280
$gameHeight = 720

Reply:
Heh, we thought of the same thing in terms of that situation
![2018-08-12_12-00-22](https://user-images.githubusercontent.com/15317421/44005376-4d786822-9e27-11e8-8490-a8c179b4d3bd.png)

Anyway it might be best to copy the presets and just put 1280x720 as the first preset for the default. That's what @Crementif was planning on doing anyways.

Reply:
How about:
 Default to native
Standard resolutions High -> Low
Unusual resolutions High -> Low